modules/loader/src/test/java/com/opengamma/strata/loader/LoaderUtilsTest.java

```diff
@@ -69,6 +69,14 @@ public void test_parseBoolean() {
   @Test
   public void test_parseInteger() {
     assertThat(LoaderUtils.parseInteger("2")).isEqualTo(2);
+    assertThat(LoaderUtils.parseInteger("(2)")).isEqualTo(-2);
+    assertThat(LoaderUtils.parseInteger("(23)")).isEqualTo(-23);
+    assertThatIllegalArgumentException()
+        .isThrownBy(() -> LoaderUtils.parseInteger("()"))
+        .withMessage("Unable to parse integer from '()'");
```

Comment:
With AssertJ you have to put the `withMessage` before the method that actually throws, otherwise the message is not picked up. Needs fixing in all the methods below.

Reply:
My mistake. This is correctly testing the message that is thrown.